haiku-misc/ssh_askpass/ssh_askpass-0.1.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A native Haiku UI for entering SSH passwords / -phrases"
+DESCRIPTION="SSH_askpass is a simple ui for authenticating your SSH connections"
+HOMEPAGE="https://github.com/ilzu/haiku_ssh_askpass"
+COPYRIGHT="2024 Ilmari \"ilzu\" Siiteri"
+LICENSE="MIT"
+REVISION=1
+SOURCE_URI="$HOMEPAGE/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="68966119a093dd5ec0446de23deee99b81b9a42bb27dee17ce687133beecfeb3"
+SOURCE_DIR="haiku_ssh_askpass-$portVersion"
+
+ARCHITECTURES="all"
+
+PROVIDES="
+	ssh_askpass = $portVersion
+	cmd:ssh_askpass
+	"
+REQUIRES="
+	haiku
+	openssh
```

Comment:
should be cmd:ssh

Reply:
Fixed in my fork of HaikuPorts, should I make another pull request for that?

Reply:
Please do, my bad here :) be sure to revbump it in the new PR.